app/Forms/UpdateProfileForm.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Forms;
+
+use Kris\LaravelFormBuilder\Form;
+use Carbon\Carbon;
+use App\Member;
+use App\College;
+use App\Course;
+use App\Scholarship;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Auth;
+use DB;
+
+class UpdateProfileForm extends Form
+{
+	private $details = null;
+	public function __construct()
+	{
+		$user = Auth::user();
+		$this->details = DB::table('members')->where('crsid', $user->crsid)->first();
```

Comment:
Probably want to relook this. At the best, this should definitely be folded into the app\User.php class so you can do something like `Auth::user()->member()` just like you can do `$member->course()`...